.travis.yml

```diff
@@ -1,12 +1,20 @@
 language: cpp
 compiler: gcc
 cache: apt
-before_install:
-  - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
-  - sudo add-apt-repository ppa:apokluda/boost1.53 -y
-  - sudo apt-get update -qq
-  - if [ "$CXX" = "g++" ]; then sudo apt-get install -qq g++-4.8; fi
+addons:
+  apt:
+    sources:
+      - boost-latest
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.8
```

Comment:
Yes, you are right. I removed the gcc entry but it is a dependency for g++. There's also no need to set the `CC` variable as it won't be used.